agent/util-scripts/pbench-add-metalog-option

```diff
@@ -1,35 +1,56 @@
 #!/usr/bin/env python3
 
-# Usage: pbench-add-metalog-option <metadata log file> <section> <option>
+import sys
+from argparse import ArgumentParser
 
-# Add an option to a section of the metadata.log file.
-# E.g. using an 'iterations' arg for the option
-#
-# iterations: 1-iter, 2-iter, 3-iter
-#
-# where the iterations are in the <iterations.file>, one iteration per line.
+from pbench.cli.agent.commands.log import add_metalog_option
 
-import sys
-from configparser import ConfigParser, NoSectionError
 
+def main(prog, parsed):
+    """main - main program entry point to add an option to, or update an option
+    in, a section of a given metadata.log file.
+
+    To create a section name "run", with an option named, "iterations", having
+    a value of "1-iter, 2-iter, 3-iter", you would do the following:
 
-def main(lfile, section, option):
-    config = ConfigParser()
-    config.read(lfile)
-    # python3
-    # config[section][option] = ', '.join(sys.stdin.read().split())
-    sin = sys.stdin.read().replace("%", "%%")
+      $ echo "1-iter 2-iter 3-iter" \
+        | pbench-add-metalog-option ./metadata.log run iterations
```

Comment:
I prefer the "here string" syntax:
```
      $ pbench-add-metalog-option ./metadata.log run iterations <<< "1-iter 2-iter 3-iter"
```

Reply:
I used the same syntax as it is used in our bench-scripts code.

Reply:
Yeah...and I've been itching to point out that that could be improved with this syntax as well!  :-D